ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/daemon_terminattr.schema.yml

```diff
@@ -210,24 +210,3 @@ keys:
         description: |
           The default compression scheme when streaming to CloudVision is gzip since TerminAttr 1.6.1 and CVP 2019.1.0.
           There is no need to change the compression scheme.
-      ingestauth_key:
-        type: str
-        description: |
```

Comment:
Should we not flag these keys as removed and point them to the new keys?

Reply:
These were removed in AVD 3.0.0 and covered by the migration feature back then. So no I think we are past that point.